.gitlab-ci.yml

```diff
@@ -131,35 +135,62 @@ test:
     matrix:
       - IMAGE: centos:centos7
         FLAVOR: [datadog-agent, datadog-dogstatsd, datadog-iot-agent]
-        MAJOR_VERSION: [6, 7]
+        MAJOR_VERSION: 7
```

Comment:
Would it be possible to move MAJOR_VERSION out of the matrix to just have the images in there?